### PR TITLE
 Add bad pixel monitor Bokeh plots

### LINKED ISSUE #588: Build Bad Pixel Monitor bokeh plots

### PR DESCRIPTION
**Pull request by bhilbert4**
Resolves #588 

This PR includes code to create plots and tables for the web app from the results of the bad pixel monitor. 

Necessary file framework only at this point. Still lots of significant updates to be made.

### CODE REVIEW

File: jwql/website/apps/jwql/monitor_pages/badpixel_monitor.py
Comment (bourque) [inline jwql/website/apps/jwql/monitor_pages/badpixel_monitor.py:1]: @bhilbert4 Is it ok to remove this file completely? Or should we leave it around until we are completely finished with this monitor?

File: jwql/website/apps/jwql/monitor_pages/yaml/badpixel_monitor_interface.yaml
Comment (bourque) [inline jwql/website/apps/jwql/monitor_pages/yaml/badpixel_monitor_interface.yaml:1]: This is quite hefty!  Nice work here.

File: jwql/website/apps/jwql/monitor_pages/monitor_badpixel_bokeh.py
Comment (bourque) [inline jwql/website/apps/jwql/monitor_pages/monitor_badpixel_bokeh.py]: Thanks for all of the in-line documentation in this module!

**Review by bourque — APPROVED** [review]
@bhilbert4 Awesome work here, it is a great first implementation!  The code all looks good to me, and it works when testing locally on my end.  So thins should be good to go pending a successful jenkins build.

**Comment by bourque** [comment]
@bhilbert4 Looks like the Jenkins build passed.  I will leave it up to you on whether or not to remove that one 'junk' file and merge when you're ready!

File: jwql/website/apps/jwql/monitor_pages/badpixel_monitor.py
Comment (bhilbert4) [reply jwql/website/apps/jwql/monitor_pages/badpixel_monitor.py:1]: This file can be deleted. Was a false start a while ago.

**Comment by bhilbert4** [comment]
Ok, I'll pull out that file and merge.

**Comment by bourque** [comment]
Thanks @bhilbert4, I'll push the button now.

**Pull request merged**
